midend/copyStructures.cpp

```diff
@@ -26,8 +26,8 @@ const IR::Node *RemoveAliases::postorder(IR::AssignmentStatement *statement) {
         return statement;
     }
 
-    ReadsWrites rw(refMap);
-    if (!rw.mayAlias(statement->left, statement->right)) {
+    ReadsWrites rw;
```

Comment:
ReadWrites is now a ResolutionContext. Does this need a fixme about recreating it again and again?